tests/integration/failoverto.tcl

```diff
@@ -0,0 +1,185 @@
+start_server {tags {"failoverto"}} {
+    set replica [srv 0 client]
```

Comment:
i think we should have 2 replicas connected in all tests (and use one of them for replica_host etc.)